packages/ext-web-components-demo/package.json

```diff
@@ -9,12 +9,12 @@
         "build": "npm run clean && cross-env webpack --env.environment=production --env.treeshake=yes --env.basehref=/"
     },
     "devDependencies": {
-        "@sencha/ext-web-components-modern": "~7.2.1",
-        "@sencha/ext-webpack-plugin": "~7.2.0",
-        "@sencha/ext": "~7.2.0",
-        "@sencha/ext-modern": "~7.2.0",
-        "@sencha/ext-modern-treegrid": "~7.2.0",
-        "@sencha/ext-modern-theme-material": "~7.2.0",
+        "@sencha/ext-web-components-modern": "~7.3.0",
+        "@sencha/ext-webpack-plugin": "~7.3.1",
```

Comment:
@ChartistDev Please be more thorough in these updates! We aren't release a patch version! Please change this.